_posts/2022-08-24-dc30-hhv-challenges.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: DEF CON 30 HHV CTF Recap
+date: 2022-08-24 12:00:00 -0000
+author: mediumrehr
+author_url: https://blog.rehr.io/
+image: assets/images/header.png
+categories: [CTF, DC30]
+---
+
+Thank you all for another great event! It was good to be back in person for DC30, and we're extremely pleased with the turnout for this year's CTF. Congratulations to elklepo for winning this year's HHV CTF. Challenge descriptions, write ups, and stats are now all posted on the [DC30 HHV Challenge Page](/challenges/dc30).
+
+If you're interested in helping create content for next year's CTF, or if you want to volunteer with the DC HHV and SSV, please join us over in our [DC HHV & SSV discord](https://discord.gg/3TdXrqy8)! We likely will have a call for CTF content creators and volunteers for DC31 sometime in 2023, but please let us know if you're interested in joining our team.
```

Comment:
Don't discord links expire?

![image](https://user-images.githubusercontent.com/5025152/186481881-813d5c43-a30f-484d-b4a5-46db97458478.png)

Reply:
good call. fixed this to be a link that doesn't expire.